### DISCUSSION TITLE
p2p for air actionscript 2022

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#1614] [General]
how can I organize a p2p connection in air using actionscript 3

previously , flash media server and NetConnection classes were used for this . At the moment, flash media server is not available, and there is no understanding how to replace it.

Tell me please.

**Comment by ajwfrost** [comment]
Adobe Media Server is still required for this -> please see Veriskope for getting hold of this, they took over the Media Server in a similar way to how we took over FP/AIR
https://www.veriskope.com/

thanks

**Reply by marchbold** [reply]
Ah awesome, that's good to know!

**Reply by skramovskiy** [reply]
спасибо за ответ.
Раньше можно было без покупки подключаться, и настраивать обмен информацией между групп. теперь только покупать так понимаю?

**Reply by ajwfrost** [reply]
With some help from Google Translate :-)

My recollection was that there was a free trial available (from Adobe) that had some limitations e.g. 20 minute max on the streams. I don't know whether Veriskope offer the same; but they'd need to be offering this on a commercial basis so I'm not sure how much you can do for free now.

**Reply by radubirsan** [reply]
https://www.monaserver.ovh/index.html

**Reply by raresn** [reply]
@radubirsan we use monaserver as well - very good for p2p stuff.
